docs/security/access-control.md

```diff
@@ -20,24 +42,90 @@ Gravitino doesn't support metadata authentication. It means that Gravitino won't
 
 :::
 
-
 ## Concept
 
+### Authorization
+
+Gravitino also provides a set of authorization frameworks to interact with different underlying data source
+authorization systems (e.g., MySQL's own permission management and the Apache Ranger permission management system for big data)
+in accordance with its own authorization model and methodology.
+More information you can see [Authorization push down](authorization-pushdown.md).
+
+### Authentication
+
+As mentioned above, Gravitino uses Ownership to control the rights of resources in the management category and uses Role to control the permissions of operations in the operation category,
+so when a user performs a specific operation on a specified resource,
+Gravitino will perform a composite authentication on the Ownership and Role to which the resource belongs.
+When a user has more than one Role, Gravitino will use the user's current Role for authentication, and the user can switch the current Role to access different resources.
```

Comment:
This part seems to relate to authorization rather than authentication.

Reply:
The authorization relates to access control.
Here's just a brief description and add jump to authorization-pushdown.md